kcidb/misc.py

```diff
@@ -12,14 +12,15 @@
 import logging
 import json
 from textwrap import indent
+
```

Comment:
Please do not do unrelated whitespace/formatting changes. If you need to change this for some reason, please make a separate commit with this change explaining the reasoning in the commit message.